libs/platform/src/lib/components/form/checkbox-group/checkbox-group.component.spec.ts

```diff
@@ -0,0 +1,451 @@
+    });
+});
+
+@Component({
+    selector: 'fdp-cbg-template-driven-test',
+    template: `
+        <fdp-form-group [multiLayout]="true">
+            <fdp-form-field #fl1 [id]="'phones'" [label]="'Phones interested in:'" zone="zLeft" rank="1">
+                <fdp-checkbox-group [list]="phoneslist" [name]="'brands'" [(ngModel)]="phones"></fdp-checkbox-group>
+            </fdp-form-field>
+
+            <fdp-form-field #fl2 [id]="'visited'" [label]="'Country visited: '" zone="zLeft" rank="1">
+                <fdp-checkbox-group
+                    [list]="countryVisited"
+                    [name]="'visited'"
+                    [(ngModel)]="visited"
+                ></fdp-checkbox-group>
+            </fdp-form-field>
+
+            <fdp-form-field #fl3 [id]="'hobbies'" [label]="'My Hobbies:'" zone="zLeft" rank="1">
+                <fdp-checkbox-group [name]="'hobby'" [(ngModel)]="hobbies">
+                    <fdp-checkbox [value]="'cooking'" [label]="'Cooking'"></fdp-checkbox>
+                    <fdp-checkbox [value]="'painting'" [label]="'Painting'"></fdp-checkbox>
+                    <fdp-checkbox [value]="'coding'" [label]="'Coding'"></fdp-checkbox>
+                    <fdp-checkbox [value]="'gardening'" [label]="'Gardening'"></fdp-checkbox>
+                </fdp-checkbox-group>
+            </fdp-form-field>
+
+            <fdp-form-field [id]="'language'" [label]="'Languages Known: '" zone="zLeft" rank="1">
+                <fdp-checkbox-group [list]="languages" [name]="'language'" [(ngModel)]="language"></fdp-checkbox-group>
+            </fdp-form-field>
+
+            <fdp-form-field [id]="'fruits'" [label]="'Fruits:'" zone="zLeft" rank="1">
+                <fdp-checkbox-group [name]="'fruits'" [(ngModel)]="fruits">
+                    <fdp-checkbox [value]="'apple'" [label]="'Apple'" [disabled]="true"></fdp-checkbox>
+                    <fdp-checkbox [value]="'banana'" [label]="'Banana'"></fdp-checkbox>
+                    <fdp-checkbox [value]="'guava'" [label]="'Guava'" [disabled]="true"></fdp-checkbox>
+                    <fdp-checkbox [value]="'papaya'" [label]="'Papaya'"></fdp-checkbox>
+                </fdp-checkbox-group>
+            </fdp-form-field>
+        </fdp-form-group>
+    `
+})
+class TestTemplateDrivenCheckboxGroupComponnet {
+    phoneslist: string[] = ['Samsung', 'Apple', 'OnePlus', 'Redmi'];
+    countryVisited = [new Country('Australia', 'Australia'), new Country('India', 'India'), new Country('USA', 'USA')];
+    phones = ['Samsung', 'OnePlus'];
+    visited = ['India', 'USA'];
+    hobbies = ['coding', 'gardening'];
+
+    fruits = '';
+    languages = [
+        new LanguageKnown('Java', 'java', false),
+        new LanguageKnown('Javascript', 'javascript', true),
+        new LanguageKnown('Python', 'python', false),
+        new LanguageKnown('GoLang', 'go', true)
+    ];
+}
+
+fdescribe('Checkbox Group Component Template driven Form Tests', () => {
```

Comment:
Change `fdescribe` to `describe`

Reply:
Done.
Thanks, had forgot to remove after debug.